packages/strapi-plugin-content-manager/admin/src/pluginRequirements.js

```diff
@@ -0,0 +1,3 @@
+export const shouldRenderCompo = (plugin) => new Promise((resolve) => {
```

Comment:
Export as default.

Reply:
The name of the file should be `requirements.js` instead of `pluginRequirements.js` in order to be consistent with `bootstrap.js`.